ctest4j-common/src/main/java/edu/illinois/Names.java

```diff
@@ -42,6 +42,12 @@ public class Names {
     public static final String PID_TID_SEPARATOR = "@";
     /** The name of maven surefire plugin */
     public static final String SUREFIRE_PLUGIN_KEY = "org.apache.maven.plugins:maven-surefire-plugin";
+    /** The method to get the value of the configuration parameter */
+    public static final String CTEST_GETTER = "ctest.getter";
```

Comment:
Can you also add these properties to [Options.md](https://github.com/xlab-uiuc/ctest4j/blob/ctest4j_aspectj/Options.md)?

Reply:
Ok, done.